presto-docs/src/main/sphinx/admin/tuning.rst

```diff
@@ -10,29 +10,392 @@ Config Properties
++                                          +                                          +                                    +may increase CPU usage for garbage collecting and use  +
++                                          +                                          +                                    +additional constant memory even if there is nothing to +
++                                          +                                          +                                    +do for all of the threads.                             +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++            ``query.max-age``             +          ``String`` (duration)           +              ``15 m``              +This property describes time after which the query     +
++                                          +                                          +                                    +metadata may be removed from server. If value is low,  +
++                                          +                                          +                                    +it's possible that client will not be able to receive  +
++                                          +                                          +                                    +information about query completion. The value          +
++                                          +                                          +                                    +describes minimum time that must pass to remove query  +
++                                          +                                          +                                    +(after it's considered completed) but if there is      +
++                                          +                                          +                                    +space available in history queue the query data will   +
++                                          +                                          +                                    +be kept longer. The size of history queue is defined   +
++                                          +                                          +                                    +by ``query.max-history`` property (``100`` by          +
++                                          +                                          +                                    +default).                                              +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++     ``query.max-concurrent-queries``     +       ``Integer`` (at least ``1``)       +              ``1000``              +**Deprecated** Describes how many queries be processed +
```

Comment:
IDK - there were on a list I was ordered to describe.
For me we can safely remove them but then we probably should extend https://prestodb.io/docs/current/admin/queue.html article (new way of defining this) to cover those possibilities.